classes/Video.php

```diff
@@ -613,12 +613,22 @@ public function getHttpResponse(array $headers = [])
     {
         $client = new Client();
         $urls = $this->getUrl();
+        $stream_context_options = [];
+
+        if (array_key_exists('Referer', $this->http_headers)) {
```

Comment:
`http_headers` is an object so you should explicitly convert it to an array:
```suggestion
        if (array_key_exists('Referer', (array)$this->http_headers)) {
```

Reply:
Thanks again!
I did the small adjustment.